ztp/kube-compare-reference/optional/ptp-config/PtpConfigDualFollower.yaml

```
@@ -0,0 +1,138 @@
+# In this example, 2 ports of a single NIC are used in a active/standby configuration to synchronize to a 
+# remote grandmaster via interfaces $iface1 and $iface1
+apiVersion: ptp.openshift.io/v1
+kind: PtpConfig
+metadata:
+  name: ordinary-clock-1
+  namespace: openshift-ptp
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "10"
+spec:
+  profile:
+  {{- range .spec.profile }}
+  - name: oc1
+    phc2sysOpts: -a -r -n 24 -N 8 -R 16 -u 0
```

```suggestion
    phc2sysOpts: -a -r -n (?<domainNumber>[0-9]+) -N 8 -R 16 -u 0
```
This `24` should be customizable, and the same as `domainNumber`, below.  By using the same named capturegroup for both we will enforce (in 4.19) that they are the same value.

---

Fixed